src/api/models/view_models/__init__.py

```
@@ -1,3 +1,4 @@
 # view_models should only create postgres views that other db entities do not depend on
 
 from .fish_attributes import *  # noqa: F403
+from .invert_attributes import *  # noqa: F403
```

Flake8 (7.3.0)

[error] '.invert_attributes.*' imported but unused

(F401)